packages/discord.js/src/structures/Guild.js

```diff
@@ -451,8 +451,12 @@ class Guild extends AnonymousGuild {
    * @param {BaseFetchOptions} [options] The options for fetching the member
    * @returns {Promise<GuildMember>}
    */
-  fetchOwner(options) {
-    return this.members.fetch({ ...options, user: this.ownerId });
+  async fetchOwner(options) {
+    if (!this.ownerId) {
+      throw new Error('FETCH_OWNER_ID');
+    }
+    const member = await this.members.fetch({ ...options, user: this.ownerId });
+    return member;
```

Comment:
Can’t you just return this without assigning a constant and remove the await? What’s the point of this variable?

Reply:
Making the linter happy that this isn't an async method without await.

Reply:
Then don’t make this an async method and return a Promise.reject

Reply:
But we prefer async methods - I checked that internally first.

Reply:
Not really, most methods return a promise rejection when they don’t use the await keyword anywhere else.